static/js/around-map.js

```diff
@@ -107,45 +111,47 @@ function urlFor (id) {
 }
 
 function findPoi (id) {
-  return pois.find(poi => poi._id == id)
+  return pois.find(poi => poi._id === id)
 }
 
 function highlight (poi_id) {
   const cardActive = document.querySelector('.current-card')
   const markerActive = document.querySelector('.current-marker')
-  const card = document.querySelector(`#card-${poi_id}`)
-  const marker = document.getElementById(poi_id)
   const superMarker = document.querySelector('.master-marker')
+  // querySelector don't works if poi_id begin with a number
```

Comment:
It must have been a headache to find that out, that's great to explain it for the future dev 👍